Utilities/src/test/java/edu/stanford/hivdb/mutations/ApobecTest.java

```diff
@@ -20,46 +20,152 @@
 
 import static org.junit.Assert.*;
 
-//import java.io.InputStream;
-//import java.util.List;
+import java.io.InputStream;
+//import java.sql.SQLException;
+import java.util.List;
 
 import org.junit.Test;
 
-//import edu.stanford.hivdb.testutils.TestMutationsFiles;
-//import edu.stanford.hivdb.testutils.TestMutationsFiles.TestMutationsProperties;
-//import edu.stanford.hivdb.utilities.MutationFileReader;
+import edu.stanford.hivdb.filetestutils.TestMutationsFiles;
+import edu.stanford.hivdb.filetestutils.TestMutationsFiles.TestMutationsProperties;
+import edu.stanford.hivdb.utilities.MutationFileReader;
 
 public class ApobecTest {
-	/*@Test
-	public void testCheckSequence() {
 
-		final InputStream testMutationsInputStream =
-				TestMutationsFiles.getTestMutationsInputStream(TestMutationsProperties.APOBEC_TEST);
+	// Initialization
+	@Test
+	public void testApobecMapPopulation() {	
+//		try {
+//			Apobec.populateApobecMaps();
+//		} catch (SQLException e) {
+//			e.printStackTrace();
+//		}
+	}
+	
+	@Test
+	public void testBasicApobecMutVerification() {
+		final Mutation apocecMut = new Mutation(Gene.RT, 239, "*");
+		final Mutation drmMut = new Mutation(Gene.PR, 30, "N");
+		assertFalse(Apobec.isApobecMutation(drmMut));
+		assertTrue(Apobec.isApobecMutation(apocecMut));
+	}
+	
+	@Test
+	public void testBasicDrmMutVerification() {
+		final Mutation drmMut = new Mutation(Gene.PR, 30, "N");
+		final Mutation apocecMut = new Mutation(Gene.RT, 239, "*");
+		assertFalse(Apobec.isApobecDRM(apocecMut));
+		assertTrue(Apobec.isApobecDRM(drmMut));
+	}
+		
+	@Test
+	public void testExhaustiveApobecMutVerification() {
+		final MutationSet apocecMuts = Apobec.getApobecMutsLU();
+		apocecMuts.forEach(mut -> {
+			assertFalse(Apobec.isApobecDRM(mut));
+			assertTrue(Apobec.isApobecMutation(mut));
+		});
+	}
+	
+	@Test
+	public void testExhaustiveDrmMutVerification() {
+		final MutationSet drms = Apobec.getApobecDRMsLU();
+		drms.forEach(mut -> {
+			assertFalse(Apobec.isApobecMutation(mut));
+			assertTrue(Apobec.isApobecDRM(mut));
+		});
+	}
+	
+	@Test
+	public void testZeroMutsAtDRP() {
+		final MutationSet nonDRPmuts = new MutationSet("PR6* PR17k PR:D30N IN:G140S RT:M230I");
+		final Apobec a = new Apobec(nonDRPmuts);
+		assertEquals(new MutationSet(""), a.getApobecMutsAtDRP());
+	}
+	
+	@Test
+	public void testMultipleMutsAtDRP() {
+		final MutationSet eMutsAtDR = new MutationSet("PR48R IN140E RT190R");
+		final Apobec a = new Apobec(eMutsAtDR);
+		assertEquals(eMutsAtDR, a.getApobecMutsAtDRP());
+	}
+		
+	@Test
+	public void testZeroMuts() {
+		final String expected = "The following 0 APOBEC muts were present in the sequence: .";
```

Comment:
This is a weird output. Just open an issue to record it since it's very low priority.